packages/contracts-ethers/CHANGELOG.md

```diff
@@ -4,8 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased v0.0.1-alpha
-
+## v0.1.0-alpha
+TODO What to write here?
+## v0.0.1-alpha
```

Comment:
@mathewmeconry What should be written here?

Reply:
there are no workflows yet to automatically create a github release with the changelog.
We can add this if you like. See the aragon/sdk repo as a reference

Reply:
Would be nice to have once all of us are used to the workflow of maintaining the changelog.